meshtastic/node_runtime/transport_runtime.py

```
@@ -0,0 +1,422 @@
+"""Admin transport, channel write/delete, ACK/NAK, and position/time command runtimes."""
+
+import logging
+import time
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Any, Callable
```

Ruff (0.15.6)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)